packages/adapter-node/src/handler.js

```diff
@@ -19,7 +19,13 @@ const address_header = env('ADDRESS_HEADER', '').toLowerCase();
 const protocol_header = env('PROTOCOL_HEADER', '').toLowerCase();
 const host_header = env('HOST_HEADER', 'host').toLowerCase();
 const port_header = env('PORT_HEADER', '').toLowerCase();
-const body_size_limit = parseInt(env('BODY_SIZE_LIMIT', '524288')) || undefined;
+const body_size_limit = parseFloat(env('BODY_SIZE_LIMIT', '524288'));
```

Comment:
actually we should use `Number` rather than `parseFloat`, which is unreasonably forgiving (it will return e.g. `0` for `'0_o'` instead of `NaN`)

```suggestion
const body_size_limit = Number(env('BODY_SIZE_LIMIT', '524288'));
```